ddtrace/_monkey.py

```diff
@@ -84,8 +84,29 @@
     "dogpile_cache": True,
     "yaaredis": True,
     "asyncpg": True,
+    "datadog_lambda": bool(os.environ.get("AWS_LAMBDA_FUNCTION_NAME", False)),  # patch only in AWS Lambda environments
```

Comment:
What's the expected behavior if datadog_lambda is patched outside of AWS Lambda environments? 

Can we move the `os.environ.get("AWS_LAMBDA_FUNCTION_NAME", False)` check to `ddtrace/contrib/datadog_lambda/patch.py:patch()`? Ideally _monkey.py module should not contain logic specific to an integration. It should just attempt to patch.

Reply:
The expected behavior is to do nothing when outside an AWS Lambda.

And thank you for the suggestion, I really didn't like adding code to `_monkey.py` like that, so know I can move the logic makes sense, thank you!